src/framework/mlt_producer.h

```diff
@@ -80,6 +80,25 @@ struct mlt_producer_s
 	 */
 	int ( *get_frame )( mlt_producer, mlt_frame_ptr, int );
 
+	/** Seek to s specified position (virtual function).
```

Comment:
spelling error: "to s" instead of "to a"

Reply:
Done